Heavy-Division/B78XH
# refactor: IRS alignment time update

# Title
refactor: IRS alignment time update 



# Description 
For the real time case value:
Added code to calculate time based on the latitude.
The math was sourced from a graph from the Embraer 170 operating manual. According to Honeywell, it has the same IRS unit as the 787. 
For ret value:
restructured the random number calculations.




# Source material
https://caisatech.net/uploads/XXI_3_EMBRAER_H51_E170_O_AOM-1&2_R21_20OCT2016.pdf
IRS alignment time graph is on page 123.
math equation for the curve between 0 and 60 is
y=1.07415^(x-37.5)+5




# Testing Instructions 
load the craft in different latitude zones and turn on the IRS alignment to see how much time remaining shows up


# Test Results
45.399 lat - Wakkanai Airport (time to align 6 min 45 sec)
65.117 lat - Pechora Airport (time to align 10 min)
74.717 lat - Resolute Bay Airport (time to align 17 min)
(Screenshots are a few seconds off because of time time to reposition camera to take screenshot after turning IRS on)



# Discord Username
vish#3861
 
![resolute](https://user-images.githubusercontent.com/105092053/172757683-5abcb2ed-717e-4f47-8121-c39d66316685.jpg)
![Wakkanai](https://user-images.githubusercontent.com/105092053/172757686-ebb3b2e4-0247-4384-b884-57eff9d42fe6.jpg)
![pechora](https://user-images.githubusercontent.com/105092053/172757688-c56d726a-0692-4d65-892e-ac8b3654d219.jpg)
![graph](https://user-images.githubusercontent.com/105092053/172759474-011ee925-6dc1-4b7f-aa62-4d925d9f5f20.png)
